kubespawner/spawner.py

```diff
@@ -1809,18 +1836,32 @@ def _expand_user_properties(self, template):
         safe_username = escapism.escape(
```

Comment:
Shall we remove this `legacy_escaped_username` now, as it is not used anymore ?

Reply:
I don't think so, since it is only unused for the opt-in new scheme.